jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -57,7 +57,7 @@
       ansible.builtin.find:
         paths: /home/{{ ansible_user_id }}
         recurse: no
-        patterns: "logs-jenkins-airship*"
+        patterns: "logs-jenkins-*"
```

Comment:
trying to understand this? What would be the impact if we add metal3

Reply:
It should be no different. The glob will match the Jenkins logs regardless of the name of the job. So my change makes it work with both the new name and the old name, so it's one fewer thing to have to transition.